.github/workflows/compliance.yml

```diff
@@ -15,6 +16,16 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
+  compliance-checks:
```

Comment:
would replace this with semantics or just prefix it with pr/pull request

Reply:
changed wording. would we still want the semantics action?